server.go

```diff
@@ -151,26 +151,18 @@ func (s *Server) Get(path string, layout *fragment.Definition, content []*fragme
 	s.routes = append(s.routes, *route)
 }
 
-func (s *Server) IgnoreHeader(name string) {
-	s.ignoreHeaders[http.CanonicalHeaderKey(name)] = true
+// target returns the configured http target
+func (s *Server) Target() string {
+	return s.target
 }
 
-func (s *Server) LoadRoutesFromFile(filePath string) error {
-	routeEntries, err := readConfigFile(filePath)
-	if err != nil {
-		return err
-	}
-
-	return s.loadRoutes(routeEntries)
+// routes returns a slice containing routes defined on the server.
+func (s *Server) Routes() []Route {
+	return s.routes
 }
 
-func (s *Server) LoadRoutesFromJSON(routesJson string) error {
-	routeEntries, err := loadJsonConfig([]byte(routesJson))
-	if err != nil {
-		return err
-	}
-
-	return s.loadRoutes(routeEntries)
+func (s *Server) IgnoreHeader(name string) {
```

Comment:
Super minor, but I wonder if we should made this `vararg` so consumers can ignore multiple headers in a single call, so take `names ...string` instead.

Reply:
That's a good idea, it should be backwards compatible so I'll go ahead and make that change.